docker/package/packages.py

```diff
@@ -189,7 +189,7 @@ def mk_node_unit(suffix, env, desc):
 common_node_env = ["NODE_RPC_ADDR=127.0.0.1:8732", "CERT_PATH=", "KEY_PATH="]
 for network in networks:
     env = [
-        f"DATA_DIR=/var/lib/tezos/node-{network}",
+        f"NODE_DATA_DIR=/var/lib/tezos/node-{network}",
```

Comment:
> The only manual steps left are 4) and 5). I think they can be added to the tezos-setup-wizard.
Alternatively, we can consider custom baking services as an advanced feature, leave these steps manual, and
document how one may start a custom baking service without the setup wizard. What do you think?

I agree with @ixahedron that the wizard shouldn't have this option because it would complicate it without being useful for the vast majority of people.

However (as she mentioned as well) this PR should add a section in the baking tutorial describing this option.

Reply:
BTW I don't think that these instructions should rely on the voting scenario nor `local-chain` in general, given that there are other ways to run custom test networks (and that `local-chain` is mostly used only by us).

Reply:
> this PR should add a section in the baking tutorial describing this option.

I propose to update the baking tutorial later once updated packages are published

Reply:
> BTW I don't think that these instructions should rely on the voting scenario nor `local-chain` in general, given that there are other ways to run custom test networks (and that `local-chain` is mostly used only by us).

Sure, instructions in the PR description are based on `local-chain` only because this is the most convenient way to test changes for us